server/@types/shared/models/OffenderDto.ts

```diff
@@ -3,7 +3,7 @@
 /* tslint:disable */
 /* eslint-disable */
 export type OffenderDto = {
-    objectType: 'Not_Found' | 'Limited' | 'Full';
     crn: string;
+    objectType: 'Not_Found' | 'Limited' | 'Full';
```

Comment:
This keeps swapping locally for me which is really annoying! hopefully will fix it